libs/auth-api-lib/src/lib/delegations/delegation-scope.service.ts

```diff
@@ -161,6 +167,47 @@ export class DelegationScopeService {
     })
   }
 
+  private findValidGeneralMandateTo(
+    toNationalId: string,
+    fromNationalId: string,
+  ): Promise<Delegation[]> {
```

Comment:
We should let this be focused on finding scopes

```suggestion
  private findValidGeneralMandateScopesTo(
    toNationalId: string,
    fromNationalId: string,
  ): Promise<String[]> {
```